code/drasil-example/Drasil/DblPendulum/Unitals.hs

```diff
@@ -49,7 +49,7 @@ lenRod = makeUCWDS "l_rod" (cn "length of rod")
         (sub cL lRod) metre
 
 pendDisplacementAngle = makeUCWDS "pendDisplacementAngle" (cn "displacement angle of pendulum")
-        (phrase angle `the_ofThe` phrase pendulum)
+        ( S "the" +:+ phraseNP ((angle `ofThe'` pendulum)))
```

Comment:
Perhaps this change shouldn't be with the Haddock-oriented changes?  Also, perhaps there should be a `the_ofThe'` combinator?  I hate to see a `S "the"` re-appear...